File: Aula_1/Livro.java
package lpoo2;

public class Livro {

    public String titulo;
    public String autor;
    public int ano;

    public void exibir() {
        System.out.println("Titulo: " + titulo);
        System.out.println("Autor: " + autor);
        System.out.println("Ano: " + ano);
    }

    public static void main(String[] args) {
        Livro L = new Livro();
        L.titulo = "Java Como Programar";
        L.autor = "Deitel";
        L.ano = 2011;
        L.exibir();
    }
}

File: Aula_1/Livro.py
class Livro:
  def __init__(self, titulo, autor, ano):
    self.titulo = titulo
    self.autor = autor
    self.ano = ano
  def exibir(self):
    print(f"Título: {self.titulo}")
    print(f"Autor: {self.autor}")
    print(f"Ano: {self.ano}")

L = Livro("Python Como Programar", "Deitel", 2011)
L.exibir()